packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistFolderNavItem.tsx

```diff
@@ -133,63 +135,83 @@ export const PlaylistFolderNavItem = (props: PlaylistFolderNavItemProps) => {
-          onClick={toggleIsExpanded}
-          onDragEnter={handleDragEnter}
-          onDragLeave={handleDragLeave}
-          onMouseEnter={handleMouseEnter}
-          onMouseLeave={handleMouseLeave}
+    <ClassNames>
+      {({ css }) => (
+        <Droppable
+          acceptedKinds={acceptedKinds}
+          onDrop={handleDrop}
+          css={{
+            transition: `background ${motion.quick}`,
+            borderRadius: 6
+          }}
+          hoverClassName={css({
+            backgroundColor: 'rgba(152, 73, 214, 0.15)'
+          })}
           disabled={isDisabled}
         >
-          <IconFolder
-            size='xs'
-            css={{ marginRight: spacing.s }}
-            fill={
-              folderHasUpdate ? color.secondary.secondary : color.neutral.n950
-            }
-          />
-          <span className={styles.folderName}>{name}</span>
-          <IconCaretRight
-            height={11}
-            width={11}
-            color='default'
-            className={cn(styles.iconCaret, {
-              [styles.iconCaretDown]: isExpanded
-            })}
-          />
-          <NavItemKebabButton
-            visible={isHovering && !isDraggingOver}
-            aria-label={messages.editFolderLabel}
-            onClick={handleClickEdit}
-            items={kebabItems}
-          />
-          <DeleteFolderConfirmationModal
-            folderId={id}
-            visible={isDeleteConfirmationOpen}
-            onCancel={toggleDeleteConfirmationOpen}
-          />
-        </LeftNavLink>
-        {isExpanded ? (
-          <ul>
-            {contents.map((content) => (
-              <PlaylistLibraryNavItem
-                key={keyExtractor(content)}
-                item={content}
-                level={level + 1}
-              />
-            ))}
-          </ul>
-        ) : null}
-      </Draggable>
-    </Droppable>
+          <Draggable id={id} text={name} kind='playlist-folder'>
+            <LeftNavLink
+              asChild
+              css={[
+                { display: 'flex', alignItems: 'center' },
+                isDraggingOver && { '& > *': { pointerEvents: 'none' } }
+              ]}
+              onClick={toggleIsExpanded}
+              onDragEnter={handleDragEnter}
+              onDragLeave={handleDragLeave}
+              onMouseEnter={handleMouseEnter}
+              onMouseLeave={handleMouseLeave}
+              disabled={isDisabled}
+            >
+              <Flex alignItems='center' w='100%' gap='xs'>
+                <IconFolder
+                  size='xs'
+                  fill={
+                    folderHasUpdate
+                      ? color.secondary.secondary
+                      : color.neutral.n950
+                  }
+                />
+                <Text size='s' css={{ flex: 1 }} ellipses>
+                  {name}
+                </Text>
+                <IconCaretRight
+                  height={11}
```

Comment:
and as much as we can stick to standard icon sizes... might be ideal to do size='2xs'


https://harmony.audius.co/?path=/docs/icons-utility-icons--documentation